### PR TITLE
Minor editing

### PR DESCRIPTION
**Pull request by Austin-Deccentric**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -6,14 +6,17 @@
 from flask import Flask, request, jsonify
 from flask_cors import CORS
 
+# loading the model
+summarizer = 'modelfile.pkl'
+model = p.load(open(summarizer, 'rb'))
```
Comment (Nedu) [inline app.py:12]: we need the model loaded before calling any route or they will fail

Comment (Austin-Deccentric) [reply app.py:12]: That is what I did.

**Review by Austin-Deccentric — COMMENTED** [review]
Added the json.load() method to Deserialize the data before loading into the model.

**Comment by Austin-Deccentric** [comment]
Please Review

File: Post_request.py [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
-        "articleUrl": "     "
-    }'
\ No newline at end of file
+url = 'http://127.0.0.1:5000/api/summarize'
+
+article = {'Data': Your article goes here}
+
+article = json.dumps(article)
+
+send_request = requests.post(url, article)
```
Comment (Austin-Deccentric) [inline Post_request.py:11]: Firstly, I created a variable with the API endpoint then provided a dictionary for loading the article.
Lastly, the article was serialized into a json object by invoking json.dumps() and the post request to the API endpoint was created.

**Pull request merged**